lab8_9/subtask_1.py

```diff
@@ -0,0 +1,9 @@
+names = ['Sophia', 'Emma', 'Ethan', 'Liam', 'Madison',
+         'Admin', 'Olivia', 'William', 'Benjamin', 'Charlotte']
+greetings = [f"{name}, I hope you're well" if name == 'Admin' else
+             f'{name}, thank you for logging in again..' for name in names]
+
+if len(greetings):
+    print('\n'.join(greetings))
+else:
+    print('We need to find some users!')
```

Comment:
use ternary operator
I'm not sure but I suppose you can write condition without len(). empty list will be converted to false by default.